diff --git a/node_modules/@radix-ui/react-popper/dist/index.js b/node_modules/@radix-ui/react-popper/dist/index.js
index 93cc745..4eb9b92 100644
--- a/node_modules/@radix-ui/react-popper/dist/index.js
+++ b/node_modules/@radix-ui/react-popper/dist/index.js
@@ -76,7 +76,7 @@ var PopperAnchor = React.forwardRef(
     const composedRefs = (0, import_react_compose_refs.useComposedRefs)(forwardedRef, ref);
     React.useEffect(() => {
       context.onAnchorChange(virtualRef?.current || ref.current);
-    });
+    }, [context.onAnchorChange]);
     return virtualRef ? null : /* @__PURE__ */ (0, import_jsx_runtime.jsx)(import_react_primitive.Primitive.div, { ...anchorProps, ref: composedRefs });
   }
 );
diff --git a/node_modules/@radix-ui/react-popper/dist/index.mjs b/node_modules/@radix-ui/react-popper/dist/index.mjs
index 3cf1db5..de67935 100644
--- a/node_modules/@radix-ui/react-popper/dist/index.mjs
+++ b/node_modules/@radix-ui/react-popper/dist/index.mjs
@@ -41,7 +41,7 @@ var PopperAnchor = React.forwardRef(
     const composedRefs = useComposedRefs(forwardedRef, ref);
     React.useEffect(() => {
       context.onAnchorChange(virtualRef?.current || ref.current);
-    });
+    }, [context.onAnchorChange]);
     return virtualRef ? null : /* @__PURE__ */ jsx(Primitive.div, { ...anchorProps, ref: composedRefs });
   }
 );